plugins/gestures.koplugin/main.lua

```diff
@@ -76,8 +76,8 @@ local gestures_list = {
     two_finger_swipe_southwest = "⇙",
     spread_gesture = _("Spread"),
     pinch_gesture = _("Pinch"),
-    rotate_cw = _("Rotate ⤸ 90°"),
-    rotate_ccw = _("Rotate ⤹ 90°"),
+    rotate_cw = _("Clockwise ⤸ 90°"),
+    rotate_ccw = _("Anti-clockwise ⤹ 90°"),
```

Comment:
I believe such phrasing was purposefully decided against for reasons of clarity, but I like I well enough.

In any case anti-clockwise must be for the British translation I'm afraid.

```suggestion
    rotate_cw = _("Rotate clockwise ⤸ 90°"),
    rotate_ccw = _("Rotate counter-clockwise ⤹ 90°"),
```